src/robotFactory.js

```diff
@@ -1,10 +1,83 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name,
+    weight,
+    coords,
+    chipVersion) {
```

Comment:
each argument on a new line
```suggestion
  constructor(
    name,
    weight,
    coords,
    chipVersion,
  ) {
```